azure-pipelines.yml

```diff
@@ -198,7 +198,7 @@ jobs:
 - job: Windows
   condition: and(succeeded(), ne(variables['Build.SourceBranch'], 'refs/heads/master'))  # skip for PR merges
   pool:
-    vmImage: 'VS2017-Win2016'
+    vmImage: 'windows-2019'
```

Comment:
One question--do the Windows builds in our wheels repo "match" well to this? In other words, this won't affect the likelihood that Windows issues slip through until they're caught in the wheels building repo?

Reply:
The wheel repo will have to be adjusted to this as well. I was trying to gather this information accurately in #13713, but it seems that the build location of the windows wheels was not correct.

In any case, this is for now intended to just be what it says on the label - a proof-of-concept.